appeals/api/src/server/openapi-types.ts

```diff
@@ -34,145 +34,140 @@ export interface ValidateDate {
+		/** @example "Somewhere" */
+		siteAddressLine1?: string;
+		/** @example "Somewhere St" */
+		siteAddressLine2?: string;
+		/** @example "SOM3 W3R" */
+		siteAddressPostcode?: string;
+		/** @example "Somewhereville" */
+		siteAddressTown?: string;
+		/** @example 22 */
+		siteAreaSquareMetres?: number;
+		/** @example ["It's dangerous"] */
+		siteSafetyDetails?: string[];
 	};
 	documents?: {
-		/** @example "4547fec8-16b2-47bb-836d-4d0baac04079" */
+		/** @example "2024-03-01T13:48:35.847Z" */
+		dateCreated?: string;
+		/** @example "001" */
 		documentId?: string;
-		/** @example "exercitation reprehenderit laborum eu enim" */
+		/** @example "appellantCostsApplication" */
+		documentType?: string;
+		/** @example "https://pinsstdocsdevukw001.blob.core.windows.net/uploads/055c2c5a-a540-4cd6-a51a-5cfd2ddc16bf/788b8a15-d392-4986-ac23-57be2f824f9c/--12345678---chrishprofilepic.jpeg" */
+		documentURI?: string;
+		/** @example "img.jpg" */
 		filename?: string;
-		/** @example "cupidatat" */
+		/** @example "image/jpeg" */
+		mime?: string;
+		/** @example "oimg.jpg" */
 		originalFilename?: string;
-		/** @example 23810727 */
+		/** @example 10293 */
 		size?: number;
-		/** @example "non cupidatat ea" */
-		mime?: string;
-		/** @example "elit est Excepteur cupidatat" */
-		documentURI?: string;
-		/** @example "1890-01-01T22:21:50.0Z" */
-		dateCreated?: string;
-		/** @example "originalApplicationForm" */
-		documentType?: string;
 	}[];
 	users?: {
-		salutation?: any;
-		/** @example "Ray" */
+		/** @example "test@test.com" */
+		emailAddress?: string;
+		/** @example "Testy" */
 		firstName?: string;
-		/** @example "Liotta" */
+		/** @example "McTest" */
 		lastName?: string;
-		/** @example "test@example.com" */
-		emailAddress?: string;
-		/** @example "Agent" */
+		salutation?: any;
+		/** @example "Appellant" */
 		serviceUserType?: string;
 	}[];
 }
 
 export interface QuestionnaireData {
-	questionnaire?: {
-		/** @example "6000526" */
+	casedata?: {
+		/** @example "6000000" */
 		caseReference?: string;
-		/** @example "Q9999" */
-		LPACode?: string;
-		/** @example true */
-		isAppealTypeAppropriate?: boolean;
-		/** @example true */
-		doesTheDevelopmentAffectTheSettingOfAListedBuilding?: boolean;
-		affectedListedBuildings?: {
-			/** @example "1021477" */
-			listEntry?: string;
-		}[];
-		/** @example true */
-		inCAOrRelatesToCA?: boolean;
-		/** @example true */
-		siteWithinGreenBelt?: boolean;
-		/** @example ["A public notice at the site","Letters to neighbours","Advert in the local press"] */
-		howYouNotifiedPeople?: string[];
-		/** @example true */
-		hasRepresentationsFromOtherParties?: boolean;
-		/** @example true */
-		doesSiteRequireInspectorAccess?: boolean;
+		/** @example "2024-05-31T23:00:00.000Z" */
+		lpaQuestionnaireSubmittedDate?: string;
+		/** @example "cupidatat ipsum eu culpa" */
+		lpaStatement?: string;
+		/** @example ["Here it is"] */
+		siteAccessDetails?: string[];
```

Comment:
Is there a reason that siteAccessDetails was changed to array? If justified, why not do the same for siteSafetyDetails?

Reply:
it is!